src/pages/home/HomePage.js

```diff
@@ -77,7 +78,7 @@ class HomePage extends PureComponent {
             }
             PersonalDetails.fetch();
             PersonalDetails.fetchTimezone();
-            getBetas();
+            User.getBetas();
```

Comment:
Let me know if `User.fetch();` should be added to this as well.

Reply:
`HomePage` no longer exists but this would be great to call `User.fetch()` in `src/libs/Navigation/AppNavigator/AuthScreens.js` where we now call `PersonalDetails.fetch()` and `PersonalDetails.fetchTimezone()`